lib/src/delegates/asset_picker_viewer_builder_delegate.dart

```diff
@@ -203,13 +211,15 @@ class DefaultAssetPickerViewerBuilderDelegate
     List<AssetEntity>? selectedAssets,
     this.previewThumbSize,
     this.specialPickerType,
+    required int maxAssets,
```

Comment:
Hmm, looks like you didn't found that you can enter the viewer directly through picked assets in the demo. TLDR this shouldn't be neither required nor non-nullable. The viewer can be access anytime whether it has assets picked.

Reply:
Correct, didn't see that option.
Changing to `int? maxAssets`.